src/lib.rs

```diff
@@ -3263,6 +3265,73 @@ fn digraph_complement(
     Ok(complement_graph)
 }
 
+fn _random_layout<Ty: EdgeType>(
+    graph: &StableGraph<PyObject, PyObject, Ty>,
+    center: Option<(f64, f64)>,
```

Comment:
Maybe?
```suggestion
    center: Option<[f64; 2]>,
```
not important but i'm commenting on this to keep things consistent since i use `[f64; 2]` on my spring-layout pr.

Reply:
Yeah I agree it'll be good to be consistent. The only reason I went with a tuple here was I wasn't certain if pyo3 handled the conversion from `(0., 0.5)` in python to `[f64; 2]` in rust. Since it's a fixed 2 element array for center the python side interface seems more pythonic as a tuple instead of a list. But I think in the rust side using `[f64; 2]` makes more sense since it has a contiguous memory layout. Let me test this locally and see what we can do here. At the worst case we can have the `#[pyfunction]` functions convert from a tuple to an array.

Reply:
I tested this locally (and the unit tests cover it too), it looks like pyo3 will do the conversion from `(0.0, 0.5)` to an `[f64; 2]` in rust. So I've made this change in: https://github.com/Qiskit/retworkx/pull/305/commits/82fe3f1afc8edb6000beb40df100e54036422f43